playground-local/nuxt.config.ts

```diff
@@ -15,12 +15,14 @@ export default defineNuxtConfig({
       token: {
         signInResponseTokenPointer: '/token/accessToken'
       },
-      sessionDataType: { id: 'string', email: 'string', name: 'string', role: 'admin | guest | account', subscriptions: "{ id: number, status: 'ACTIVE' | 'INACTIVE' }[]" }
+      session: {
+        dataType: { id: 'string', email: 'string', name: 'string', role: 'admin | guest | account', subscriptions: "{ id: number, status: 'ACTIVE' | 'INACTIVE' }[]" },
+        dataResponsePointer: '/data/user'
```

Comment:
@valh1996 Interestingly enough, the tests caught some issue with your PR. I believe that this configuration should be just
```suggestion
        dataResponsePointer: '/'
```

Reply:
Confirmed. Running `pnpm test:e2e` inside `playground-local` with this configuration fixed, all the tests pass.

Reply:
@phoenix-ru Well done! An oversight on my part indeed